dashboard/config/linux/bits/android-5.10.yml

```diff
@@ -32,4 +34,5 @@ config:
 
  # Android has back-ported the KASAN_STACK_ENABLE => KASAN_STACK rename from v5.11
  - KASAN_STACK_ENABLE: [n, override]
- - KASAN_STACK: [override]
+ # Doesn't boot on Cuttlefish with KASAN_STACK
+ - KASAN_STACK: [override, -virtual_modules]
```

Comment:
I assume this is due to some boot time bug being caught by the instrumentation.

Maybe add a TODO to re-enable `KASAN_STACK` once the bug is fixed?

Reply:
Makes sense. Done